bob-common/src/configs/node.rs

```diff
@@ -476,9 +475,9 @@ pub struct Node {
     metrics: Option<MetricsConfig>,
 
     #[serde(skip)]
-    bind_ref: RefCell<String>,
+    bind_ref: Arc<Mutex<String>>,
```

Comment:
Why this was changed?

Reply:
This structure is now shared between multiple threads, so we need to use thread-safe abstractions